versions/3.0.md

```diff
@@ -786,9 +734,89 @@ schema:
   type: file
 ```
 
+#### <a name="requestBodyObject"></a>Request Body Object
+
+Describes a single request body.
+
+If the `requestBody` attribute is not present, or `schema` is not defined, the request body is not required.
+
+The serialization of the request body should be specified by the `consumes` attribute.
```

Comment:
`should`? Perhaps `MUST` - see #266. If we stick with `should`, it should be written as `SHOULD`.